src/utils.ts

```diff
@@ -116,7 +119,7 @@ async function renameFileIfNecessary(filePath: filePath, newFileName: fileName):
     }
 
     await rename(resolvedFilePath, newFilePath);
-    logger.info(`Renamed ${basename(resolvedFilePath)} to ${newFileName}${fileExtension}`);
+    logger.info(`Renamed ${basename(resolvedFilePath)} to ${newFileName}`);
```

Comment:
what is the reason to change this?

Reply:
It was missed in one of the previous PR where we updated the issue of showing the file extension two times. Now the updated newFileName contains the correct extension with it so we donot need to show the fileExtension explictly